open-irc-links/README.md

```diff
@@ -0,0 +1,13 @@
+# open-irc-links
+
+
+## What it does
+
+This add-on sets default client for opening IRC links using protocol_handlers.
+
+Whenever any link with IRC protocol is clicked it opens with pre-defined URI template.
+For example click [here](irc://irc.freenode.net/drupal).
+
+## What it shows
+
+* usage of protocol_handlers in webextension.
```

Comment:
Please change to "How to use protocol handlers to pass content to an application or website designed to handle that content."